### PR TITLE
Release/2.2.0

### PR DESCRIPTION
**Pull request by richpsharp**
This is just a change to history and that's it!

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Release History
 ===============
 
-Unreleased Changes
+2.2.0 (2020-05-04)
```
Comment (beckyck) [inline HISTORY.rst]: Hey it's the 14th, my friend. But may the fourth be with you. :)

**Review by beckyck — CHANGES_REQUESTED** [review]
Looks good except for the date!

File: HISTORY.rst (hunk 1 above)
Comment (richpsharp) [reply HISTORY.rst]: oops! fixed in 8b101f0

**Comment by richpsharp** [comment]
> Looks good except for the date!

I fixed the date! Can you look again?

**Review by beckyck — APPROVED** [review]
Cool beans!

**Pull request merged**